packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -913,7 +921,18 @@ export default function ExperimentHeader({
           setEditInfoFocusSelector={setEditInfoFocusSelector}
           editTags={!viewingOldPhase ? editTags : undefined}
         />
+
+        {experiment.status === "running" && runningExperimentStatus && (
```

Comment:
question: `RunningExperimentDecisionBanner` can return `null` in certain conditions, meaning that there's a chance we can render an empty Box here, which I think is okay because it is not styled?

Or it might create weird spacing because of the padding.

Any way we can consolidate the checks?

Reply:
I kind of took the lazy way out here to still let the component return `null` but I wanted to do that to keep the arguments simple. But made an adjustment so we shouldnt return an empty box ever, thanks!